File: java/program2.java
import java.util.*;
import java.util.Scanner;
class main
{
    public static void main(String arg[])
    {
       Scanner sc=new Scanner (System.in);
       System.out.println("Enter your name=");
       int No1=sc.nextInt();
       int Ans=No1*No1*No1;
       System.out.println("cube="+Ans);
    }
}

File: c++/program2.c
#include<Stdio.h>

long int CalculateCube(int nValue)
{

   long int Cube =0;
   Cube=nValue*nValue*nValue;

   return Cube;
}



int main() 
{
  auto int no1=0;
  auto long int Ans=0;

    printf("Enter number to cube=");
    scanf("%d",&no1);

   Ans=CalculateCube(no1);

     printf("Cube is= %d",Ans);

  return 0;
}
